docs/advanced/analytics/legacy.mdx

```diff
@@ -1,198 +1,19 @@
 ---
 sidebar_position: 6
-title: Analytics
-description:
-  Most e-commerce websites need advanced analytics to understand their users
-  better and adapt their shops to their customers' needs. But it can often be
-  tedious to maintain when you have many trackings to manage. So here is how we
-  integrated analytics services in Front-Commerce.
+title: Legacy Usage
```

Comment:
**nitpick:**  I'm afraid that `legacy` makes people a bit afraid

**suggestion:**  mention the version in introduction and (optional) in the title
```suggestion
title: Legacy Usage (< 2.19)
```